### PR TITLE
Season 6 - New weapons

### PR DESCRIPTION
**Pull request by LTKort**
No challenge data / images available yet. 
I will update them as soon as they are available, right now the new weapons are marked as Coming Soon.
See issue #106

### CODE REVIEW

**Comment by LTKort** [comment]
@carlssonemil  I added the new challenges

File: package-lock.json
Comment (carlssonemil) [inline package-lock.json:1]: This file should not be commited 🤔

**Review by carlssonemil — CHANGES_REQUESTED** [review]
Nice! Thanks! 🎉 

Remove the `package-lock.json` file from the PR and I'll merge it! 🚀

File: package-lock.json
Comment (LTKort) [reply package-lock.json:1]: Fixed in #107

**Pull request closed without merging**